upgrade_tests/helpers/common.py

```diff
@@ -68,7 +69,13 @@ def test_anyentity(pre, post)
         return assert_templates(template, pre, post)
     else:
         if isinstance(pre, list) and isinstance(post, list):
-            return sorted(list(pre)) == sorted(list(post))
+            try:
+                return sorted(list(pre)) == sorted(list(post))
+            except TypeError as err:
+                pprint('Sorted module is not supported in python3 for dict to dict '
+                       'comparison and gets error like: {}'.format(err))
+                return pre.sort(key=lambda item: item.get("id")) == \
```

Comment:
Does this works in case pre / post is type of list ?

Reply:
Yes, it worked.

Reply:
I have shared the test result in my previous comment.